tests/test_searcher.py

```diff
@@ -339,9 +345,13 @@ def test_quiescence_depth_1_alpha(
 
 
 @pytest.fixture
-def _init_searcher(max_depth: int = 4) -> Searcher:
+def _init_searcher(
```

Comment:
Re this: I have no idea how to use this properly. I can't call the function so I can't pass in arguments. Why is this a fixture instead of a pure function? I might be doing something wrong.